utils/IconSet.store.ts

```diff
@@ -370,8 +371,8 @@ export const iconSetHelper = {
           ) {
             const icon = eosIconsState.iconsCategory[i].icons[j]
             if (
-              icon!.name.includes(words![k].toLowerCase()) ||
-              icon!.tags.includes(words![k].toLowerCase())
+              icon?.name.includes(words![k].toLowerCase()) ||
```

Comment:
`words!` is better as `words?.[i]?.toLowerCase()`

Reply:
Hi, I have formatted the code to address the issue mentioned above. Please review.